ctapipe/io/hessioeventsource.py

```diff
@@ -165,6 +165,11 @@ def _generator(self):
                                                        get_azimuth_cor(tel_id))
                     data.mc.tel[tel_id].altitude_cor = (file.
                                                         get_altitude_cor(tel_id))
+                    data.pointing[tel_id].altitude = (data.mc.tel[tel_id].altitude_raw
+                                                      - data.mc.tel[tel_id].altitude_cor) * u.rad
```

Comment:
Are you sure that `altitude_cor` is the the *correction for the altitude* and not the *corrected* altitude?

Reply:
Also, you should use `u.Quantity(..., u.rad)`, it's faster than multiplying the unit.

Reply:
See line 1018ff of `io_hess.h`:

```
struct hess_tracking_event_data_struct
{
   int tel_id;            ///< The telescope ID number (1 ... n)
   double azimuth_raw;    ///< Raw azimuth angle [radians from N->E].
   double altitude_raw;   ///< Raw altitude angle [radians].
   double azimuth_cor;    ///< Azimuth corrected for pointing errors.
   double altitude_cor;   ///< Azimuth corrected for pointing errors.
   int raw_known;         ///< Set if raw angles are known.
   int cor_known;         ///< Set if corrected angles are known.
};
```

Reply:
I remember that we set it to zero to be compatbible in the simteleventsource to be compatible with the hessioeventsource, but I think it would be better to use `NaN` if it is not available at all.

Reply:
Ha. Thanks for the info.
I'll propose some corrections then.

Reply:
See that copy paste error in the comment ;-)